MantidQt/SliceViewer/src/SliceViewer.cpp

```diff
@@ -123,6 +123,14 @@ SliceViewer::SliceViewer(QWidget *parent)
   // hide unused buttons
   ui.btnZoom->hide(); // hidden for a long time
 
+  // If live data being read, save the name of the workspace(s)
+  m_liveDataWSNames.clear();
+  auto liveAlgorithms =
+      AlgorithmManager::Instance().runningInstancesOf("MonitorLiveData");
```

Comment:
I'm not sure about this. Won't this change the behaviour of all plots if we are running this algorithm? This should be a workspace dependent thing.